tests/integration/docker/exec.bats

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#Based on docker commands
+
+setup() {
+	load common
+	cleanDockerPs
+	runtimeDocker
+}
+
+@test "modifying a container with exec" {
+	# This problem is related with https://github.com/01org/cc-oci-runtime/issues/18
+	skip "exec is still not working"
+	$DOCKER_EXE run --name containertest -d ubuntu bash -c "sleep 50"
+	$DOCKER_EXE ps -a | grep "sleep 50"
+	$DOCKER_EXE exec -d containertest bash -c "echo 'hello world' > file"
+	$DOCKER_EXE exec containertest bash -c "cat /file"
+}
+
+@test "exec a container with privileges" {
+	# This problem is related with https://github.com/01org/cc-oci-runtime/issues/18
+	skip "exec is still not working"
+	$DOCKER_EXE run -d --name containertest ubuntu bash -c "sleep 30"
+	$DOCKER_EXE exec -i --privileged containertest bash -c "mount -t tmpfs none /mnt"
+	$DOCKER_EXE exec containertest bash -c "df -h | grep "/mnt""
+}
+
+@test "copying file from host to container using exec" {
+	# This problem is related with https://github.com/01org/cc-oci-runtime/issues/18
+	skip "exec is still not working"
+	content="hello world"
+	echo "$content" > file.txt
+	$DOCKER_EXE run --name containertest -d ubuntu bash -c "sleep 30"
+	$DOCKER_EXE exec -i containertest bash -c "cat > /home/file.txt " < ./file.txt
+	$DOCKER_EXE exec -i containertest bash -c "cat /home/file.txt" | grep "$content"
+	rm -rf file.txt
+}
+
+@test "stdout/stderr properly forwarded using exec" {
+	# This problem is related with https://github.com/01org/cc-oci-runtime/issues/18
+	# and also with https://github.com/01org/cc-oci-runtime/issues/171
+	skip "exec is still not working"
+	$DOCKER_EXE run --name containertest -d ubuntu bash -c "sleep 30"
+	run $($DOCKER_EXE exec containertest ls -l /etc/resolv.conf /etc/foo 2>/dev/null)
```

Comment:
That doesn't look correct. The goal here to to make sure we send stdout and stderr as distinct streams. We ls both a file that exists (generate the name of the file on stdout) and a file that doesn't (generate an error message on stderr) and we want to pipe either stdout or stderr to /dev/null to make sure we receive both stdout/err as 2 separate streams (and not both on stdout like today).

The correct tests/outputs:

```
$ docker run -i --runtime runc busybox ls -1 /etc/resolv.conf /etc/foo 2>/dev/null
/etc/resolv.conf
```

and 

```
$ docker run -i --runtime runc busybox ls -1 /etc/resolv.conf /etc/foo >/dev/null
ls: /etc/foo: No such file or directory
```